### PR TITLE
Issue#273   increase char allowance for event name

### LINKED ISSUE #273: Increase character allowance for 'event name' to 150
Increase character limit for 'event name' + display remaining characters upon typing

### PR DESCRIPTION
**Pull request by pro-Nate**
fix #273

### CODE REVIEW

File: py/shared/utils.py [hunk 1]
```diff
@@ -30,11 +30,12 @@ class Currencies(str, Enum):
 }
 
 
-def slugify(title):
+def slugify(title, max_length: int = 255):
```
Comment (samuelcolvin) [inline py/shared/utils.py:33]:
```suggestion
def slugify(title, max_length: Optional[int] = None):
```
and only truncate when `max_length is not None`, also you should use [`shorten`](https://docs.python.org/3/library/textwrap.html) to cut at a word boundary.

**Review by samuelcolvin — COMMENTED** [review]
Hard to review completely since this includes changes form other PRs, please rebase to master

File: py/shared/utils.py (hunk 1 above)
Comment (pro-Nate) [reply py/shared/utils.py]: Good idea. Done

**Review by samuelcolvin — APPROVED** [review]
👍

**Pull request merged**